apps/arm/src/main.cpp

```diff
@@ -186,52 +186,45 @@ void txCANProcessor() {
 
   while (true) {
     MotionReport report;
-    float angle = 0, speed = 0;
 
     txMsg.setID(HWBRIDGE::CANID::REPORT_TURNTABLE_MOTION);
-    Turntable::manager.getActiveController()->reportAngleDeg(angle);  // TODO: Add error handling
-    Turntable::manager.getActiveController()->reportAngularVelocityDegPerSec(speed);
-    report = {angle, speed};
+    report = {Turntable::manager.getActiveController()->reportAngleDeg(),
```

Comment:
is this dangerous since the struct is packed? does it not matter

Reply:
imo you should explicitly pack it

Reply:
what do you mean by explicitly pack it?